test/new-e2e/agent-subcommands/status_test.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package sharedcomponents
```

Comment:
suggestion: with the updated directory structure, should this be subcommands?

Reply:
Nice catch, done in 8a61ac0d4c6c4a7e38850e5f0bf5dbe26965fd3a